pkg/systemstatsmonitor/system_stats_monitor.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package systemstatsmonitor
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	"github.com/golang/glog"
+	"k8s.io/node-problem-detector/pkg/problemdaemon"
+	ssmtypes "k8s.io/node-problem-detector/pkg/systemstatsmonitor/types"
+	"k8s.io/node-problem-detector/pkg/types"
+	"k8s.io/node-problem-detector/pkg/util/tomb"
+)
+
+func init() {
+	problemdaemon.Register("system-stats-monitor", types.ProblemDaemonHandler{
+		CreateProblemDaemonOrDie: NewSystemStatsMonitorOrDie,
+		CmdOptionDescription:     "Set to true to enable, or any other value to disable."})
```

Comment:
Should this be: `Set to config file paths`?

Reply:
Good catch, thanks!
Done.